### PR TITLE
Prepare release 1.1.3

### PR DESCRIPTION
**Pull request by christianbader**

### CODE REVIEW

File: blackbox/docs/release_notes/1.1.3.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   This now raises a proper error stating that it's not supported.
+
+ - Wrong results were returned from queries with more than one level of nested
+   subselects.
+
+ - ``ORDER BY`` on joins caused incorrect order of values when having multiple
+   non-distinct values on the left part of the join. Now ``ORDER BY`` is
+   correctly applied.
+
+ - Fix: Usage of ``DISTINCT`` in a query with a ``GROUP BY`` was producing
```
Comment (joemoe) [inline blackbox/docs/release_notes/1.1.3.txt]: Why do we prefix this and the following fix with "Fix:" we don't do this for others.

File: blackbox/docs/release_notes/1.1.3.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Released on 2017/05/09.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.0.2` or higher before you upgrade to 1.1.3.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be :ref:`version_1.1.2`. Any upgrade from a version
```
Comment (Autophagy) [inline blackbox/docs/release_notes/1.1.3.txt]: Is this true? I was under the impression that for patch releases a cluster upgrade would be available from the first version in the feature version (1.1.1 in this case).

File: blackbox/docs/release_notes/1.1.3.txt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+ - Admin UI improvements.
+
+ - Improved the accuracy of results if arithmetic operators +,-,*,/ are used
+   in expressions that contain only float-type values.
+
+ - Fixed ``COPY FROM`` to be able to copy data into a partitioned table with a
+   generated column as both the primary key and the partitioned column.
+
+ - ``NullPointerException`` thrown when selecting ``routing_hash_function``
```
Comment (Autophagy) [inline blackbox/docs/release_notes/1.1.3.txt]: I think this needs to be reworded as this makes it sound as if we *now* throw a NPE when selecting from sys.shards - this fix was to fix this from happening.

File: blackbox/docs/release_notes/1.1.3.txt (hunk 1 above)
Comment (christianbader) [inline blackbox/docs/release_notes/1.1.3.txt]: Maybe we should think about a guideline about writing changelog entries?

File: blackbox/docs/release_notes/1.1.3.txt (hunk 3 above)
Comment (Autophagy) [inline blackbox/docs/release_notes/1.1.3.txt]: An example of a better release note would be the one at https://github.com/crate/crate/blob/029e99c9f23da07d956f297736edabefecb8876a/blackbox/docs/release_notes/1.1.3.txt#L53

File: blackbox/docs/release_notes/1.1.3.txt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+   non-distinct values on the left part of the join. Now ``ORDER BY`` is
+   correctly applied.
+
+ - Fix: Usage of ``DISTINCT`` in a query with a ``GROUP BY`` was producing
+   wrong results and  was changed to throw UnsupportedOperationException.
+   E.g.:
+
+       SELECT DISTINCT col1 FROM t1 GROUP BY col2, col1
+
+ - Fix: An error was thrown when using ``ORDER BY COUNT(*)`` on ``JOINS``.
```
Comment (Autophagy) [inline blackbox/docs/release_notes/1.1.3.txt]: Remove Fix: prefix here

File: blackbox/docs/release_notes/1.1.3.txt (hunk 2 above)
Comment (joemoe) [inline blackbox/docs/release_notes/1.1.3.txt]: i agree. not sure if we really need this. imho an upgrade should work from everything in 1.0.x. you need to clarify this.

**Pull request merged**